docs/csharp/programming-guide/concepts/expression-trees/index.md

```diff
@@ -104,7 +104,7 @@ Console.WriteLine(factorial);
 // Prints 120.  
 ```  
   
- For more information, see [Generating Dynamic Methods with Expression Trees in Visual Studio 2010](http://go.microsoft.com/fwlink/?LinkId=169513), which also applies to later versions of Visual Studio.
+ For more information, see [Expression Trees (C#)](../../../../csharp/programming-guide/concepts/expression-trees/index.md), which also applies to later versions of Visual Studio.
```

Comment:
I mean there is nothing else to point to "for more information" if this is all of the information.